swagger.yaml

```diff
@@ -0,0 +1,124 @@
+swagger: "2.0"
+info:
+  description: "Authors Haven - A social platform for the creative at heart."
+  version: "1.0.0"
+  title: "Authors Haven"
+  license:
+    name: "MIT"
+    url: "https://www.apache.org/licenses/LICENSE-2.0.html"
+host: "kingsmen-ah-backend-staging.herokuapp.com"
+basePath: "/api"
+tags:
+- name: "user"
+  description: "User operations"
+schemes:
+  - "https"
+  - "http"
+paths:
+  /users:
+    post:
+      tags:
+      - "user"
+      summary: "Create user"
+      description: "Creates a new user in the application"
+      operationId: "createUser"
+      produces:
+      - "application/json"
+      - "application/xml"
+      parameters:
+      - in: "body"
```

Comment:
please revert to what you had before and use `"formData"` instead of query

Reply:
Thank you @fidelisojeah. Changes noted